src/interpreter/ByteCode.h

```diff
@@ -35,56 +35,167 @@ class FunctionType;
-    F(TableGrow)                \
-    F(TableSize)                \
-    F(TableFill)                \
-    F(RefFunc)                  \
-    F(MoveI32)                  \
-    F(MoveF32)                  \
-    F(MoveI64)                  \
-    F(MoveF64)                  \
-    F(MoveV128)                 \
-    F(I32ReinterpretF32)        \
-    F(I64ReinterpretF64)        \
-    F(F32ReinterpretI32)        \
-    F(F64ReinterpretI64)        \
-    F(Jump)                     \
-    F(JumpIfTrue)               \
-    F(JumpIfFalse)              \
-    F(GlobalGet32)              \
-    F(GlobalGet64)              \
-    F(GlobalGet128)             \
-    F(GlobalSet32)              \
-    F(GlobalSet64)              \
-    F(GlobalSet128)             \
-    F(Const32)                  \
-    F(Const64)                  \
-    F(Const128)                 \
-    F(Load32)                   \
-    F(Load64)                   \
-    F(Store32)                  \
-    F(Store64)                  \
-    F(FillOpcodeTable)
+#define FOR_EACH_BYTECODE_OP(F)     \
+    F_NOP(F)                        \
+    F(Unreachable)                  \
+    F(Throw)                        \
+    F(End)                          \
+    F(BrTable)                      \
+    F(Call)                         \
+    F(CallIndirect)                 \
+    F(Select)                       \
+    F(MemorySize)                   \
+    F(MemoryGrow)                   \
+    F(MemoryInit)                   \
+    F(DataDrop)                     \
+    F(MemoryCopy)                   \
+    F(MemoryFill)                   \
+    F(TableInit)                    \
+    F(ElemDrop)                     \
+    F(TableCopy)                    \
+    F(TableGet)                     \
+    F(TableSet)                     \
+    F(TableGrow)                    \
+    F(TableSize)                    \
+    F(TableFill)                    \
+    F(RefFunc)                      \
+    F(MoveI32)                      \
+    F(MoveF32)                      \
+    F(MoveI64)                      \
+    F(MoveF64)                      \
+    F(MoveV128)                     \
+    F(I32ReinterpretF32)            \
+    F(I64ReinterpretF64)            \
+    F(F32ReinterpretI32)            \
+    F(F64ReinterpretI64)            \
+    F(Jump)                         \
+    F(JumpIfTrue)                   \
+    F(JumpIfFalse)                  \
+    F(GlobalGet32)                  \
+    F(GlobalGet64)                  \
+    F(GlobalGet128)                 \
+    F(GlobalSet32)                  \
+    F(GlobalSet64)                  \
+    F(GlobalSet128)                 \
+    F(Const32)                      \
+    F(Const64)                      \
+    F(Const128)                     \
+    F(Load32)                       \
+    F(Load64)                       \
+    F(Store32)                      \
+    F(Store64)                      \
+    F(FillOpcodeTable)              \
```

Comment:
FillOpcodeTable should be the last. Anyway, I would put these into a separate group.